### ISSUE TITLE
The UCF Epick

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#2224]

**Comment by Captainkirkdawson** [comment]
@Sherlock21 @edickens @SteveBiggs All of the code for UCF has been deployed to test3; test3 has had its ucf data refreshed. It would be most helpful if you have the time and are not wet to test the code both from a transcriber input and correction perspective AS WELL as from a user search perspective.
The rules for matches are in https://app.zenhub.com/workspaces/freereg-5c8d31ec9dfa726d3bf773e3/issues/freeukgen/myopicvicar/1135

**Comment by edickens** [comment]
Tried searching for HANNA in Kirk's file with Family member ON
![Capture.JPG](https://images.zenhubusercontent.com/5cb4c5e2cccdd751e9e85870/37c620c2-3a06-4728-ad6a-e01b4b5b9796)
I expected to find the last entry with mothers surname HAN_{1,2}
Should I have found it?

**Comment by Captainkirkdawson** [comment]
@edickens Yes and mine did
![hanna](https://user-images.githubusercontent.com/2874694/74666906-9a424a00-515f-11ea-85ea-5296b5b6f02d.JPG)

**Comment by edickens** [comment]
Of course.  The list shows the main person in the entry, not who you are searching for if you asked for Family Member.  TY
Looks to work OK.

**Comment by stsccfr** [comment]
Closing as implemented.
